<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>739</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FEFU Слово 2025</string>
  </property>
  <widget class="QWidget" name="MainArea">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="ToolPanel">
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="RandomBullshitLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="LinkButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::InsertLink"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ImageButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::CameraPhoto"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="LAlignButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::FormatJustifyLeft"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="CAlignButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="RAlignButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::FormatJustifyRight"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="WAlignButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../WAlignButton.png</normaloff>../WAlignButton.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="ChangeText">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ChangeBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ChangeMargin">
            <property name="text">
             <string>Margin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="LineSpace">
            <property name="text">
             <string>LineSpace</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="FontLayout">
        <item>
         <widget class="QFontComboBox" name="FontBox">
          <property name="writingSystem">
           <enum>QFontDatabase::WritingSystem::Latin</enum>
          </property>
          <property name="fontFilters">
           <set>QFontComboBox::FontFilter::ScalableFonts</set>
          </property>
          <property name="currentFont">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FontSize">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="9">
       <layout class="QHBoxLayout" name="TextFormatLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="BoldStyleButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ItalicStyleButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="UnderlineStyleButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::FormatTextUnderline"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditClear"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QScrollArea" name="DocumentArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <widget class="QWidget" name="DocumentContent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>984</width>
         <height>1153</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="CustomTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>802</width>
            <height>1131</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid</string>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::LineWrapMode::FixedPixelWidth</enum>
          </property>
          <property name="lineWrapColumnOrWidth">
           <number>800</number>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="createDocAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вставка</string>
    </property>
    <addaction name="imageAction"/>
    <addaction name="linkAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createDocAction">
   <property name="text">
    <string>Создать новый Файл</string>
   </property>
  </action>
  <action name="imageAction">
   <property name="text">
    <string>Вставить Изображение</string>
   </property>
  </action>
  <action name="linkAction">
   <property name="text">
    <string>Вставить Ссылку</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTextEdit</class>
   <extends>QTextEdit</extends>
   <header>customtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
